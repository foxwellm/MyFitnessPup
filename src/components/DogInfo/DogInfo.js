import React from 'react'

export const DogInfo = ({ dogFromStore: { id, name, age, gender, contact, description, url, status, distance, photo } }) => {
  const dogName = !name || !name.match(' ') ? name : name.split(' ')[0]
  return (
    <div className='DogInfo-container'>
      <div className='doginfo-img-container'>
        <img className='doginfo-img' alt='dog from shelter' src={photo}></img>
      </div>
      <div className='doginfo-info-container'>
        <div className='doginfo-name'>{dogName}</div>
        <div className='doginfo doginfo-age'>Age: {age}</div>
        <div className='doginfo doginfo-gender'>Gender: {gender}</div>
        <div className='doginfo doginfo-distance'>Location: {distance} away</div>
        <div className='doginfo doginfo-status'>Status: {status}</div>
        <div className='doginfo doginfo-contact'>Contact: {contact}</div>
        <div className='doginfo doginfo-url'>PetFinder URL: {url}</div>
        <div className='doginfo doginfo-description'>Description: {description}</div>
      </div>
    </div>
  )
}

export default DogInfo