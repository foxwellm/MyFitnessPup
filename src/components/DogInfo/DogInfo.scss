.DogInfo-container {
  display: flex;
  width: 90vw;
  height: 40vw;
  margin: 50px auto;
  text-align: left;
}

.doginfo-img-container {
  min-width: 300px;
  min-height: 450px;
  width: 30vw;
  height: 40vw;
}

.doginfo-info-container {
  width: 55vw;
  margin: 2vw 0 2vw 5vw;
}

.doginfo-img {
  width: 100%;
  height: 100%;
}

.doginfo {
  font-size: 1.5vw;
  padding: 0.2vw;
}

.doginfo-name {
  font-size: 5vw;
}
