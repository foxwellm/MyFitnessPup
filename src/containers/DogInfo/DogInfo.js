import React, {Component} from 'react'

export class DogInfo extends Component {
  render() {
    return (
      <div>
        Dog Info
      </div>
    )
  }
}

export default DogInfo