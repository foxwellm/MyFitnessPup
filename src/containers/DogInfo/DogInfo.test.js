import React from 'react'
import { shallow } from 'enzyme'
import DogInfo from './DogInfo'

describe('DogInfo', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<DogInfo />)
    expect(wrapper).toMatchSnapshot()
  })
})