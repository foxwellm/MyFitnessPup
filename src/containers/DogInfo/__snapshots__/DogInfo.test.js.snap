// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogInfo should have match the correct snapshot 1`] = `
<div>
  Dog Info
</div>
`;
